moped-database/metadata/tables.yaml

```diff
@@ -3533,45 +3536,70 @@
           columns: '*'
         update:
           columns:
-            - capitally_funded
-            - is_deleted
-            - end_date
-            - added_by
-            - milestone_id
             - project_id
-            - project_importance
-            - project_length
-            - project_order
+            - work_assignment_id
+            - end_date
+            - parent_project_id
             - project_sponsor
             - status_id
+            - project_order
+            - project_name
+            - project_length
+            - capitally_funded
+            - project_description_public
+            - current_status
             - timeline_id
+            - fiscal_year
+            - project_lead_id
+            - added_by
+            - project_uuid
+            - milestone_id
+            - project_description
             - task_order
-            - ecapris_subproject_id
             - current_phase
             - current_phase_id
-            - current_status
-            - fiscal_year
-            - knack_project_id
-            - project_description
-            - project_description_public
-            - project_name
-            - project_priority
+            - project_importance
             - project_website
-            - work_assignment_id
+            - is_deleted
+            - knack_project_id
             - date_added
-            - updated_at
-            - project_uuid
-            - parent_project_id
+            - project_priority
+            - interim_project_id
+            - ecapris_subproject_id
       retry_conf:
         interval_sec: 10
         num_retries: 0
         timeout_sec: 60
-      webhook_from_env: MOPED_API_EVENTS_URL
+      webhook_from_env: HASURA_ENDPOINT
```

Comment:
This needs to be updated in our test evironment, staging environment and prod environments

Reply:
adding to staging and prod involves updating the task definitions in AWS, in test see this PR https://github.com/cityofaustin/atd-moped/pull/887/files